.github/workflows/validate_python.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Setup python3
       uses: actions/setup-python@v5
       with:
-        python-version: '3.8.x'
+        python-version: '3.13.x'
```

Comment:
Please don't upgrade to 3.13 yet. The rest of our build pipelines are using 3.11 as documented here: https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/65903/Python-Getting-Started 

(It's okay to also run tests with older/newer versions, of course.)

Consider upgrading Python in a separate PR.

Cc: @mshafer-NI